gobcore/model/gobmodel.json

```diff
@@ -1889,6 +1889,58 @@
             "description": "De datum waarop het brondocument is opgeslagen in het register."
           }
         }
+      },
+      "onderzoek": {
+        "version": "0.1",
+        "abbreviation": "OZK",
+        "entity_id": "identificatie",
+        "has_states": true,
+        "attributes": {
```

Comment:
Ik vraag me af of in_onderzoek states heeft.

Reply:
Heb de bron en de import definitie/mapping er even bijgepakt, dit zijn inderdaad states.